libraries/commerce/favorites/subscriptions/spec.js

```diff
@@ -1,33 +1,77 @@
 import {
-  getSubscriptionCount, invoke, resetSubscriptions, subscribe,
+  subscribe,
```

Comment:
Some tests are failing because of updated subscriptions.

Reply:
done - also removed the max. favorites limit from the tests, because there's no limit anymore.